### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by meglio** [jaredreich/pell#222]
Hello. I can see no updates happening in 4 years - just curious if it got abandoned?

**Comment by VaelVictus** [comment]
Yeah and Jared's ignored any emails/pings/PRs, just [use tiptap](https://tiptap.dev/).

**Comment by chinchillas** [comment]
Too bad. It was such a fun project.

**Comment by bacloud23** [comment]
@VaelVictus 
It compares with no way with pell
https://bundlephobia.com/package/tiptap@1.32.2

**Comment by VaelVictus** [comment]
@bacloud23 Fake news! Tiptap's core is 58kb and you add plugins after, which are listed on bundlephobia as well. You also linked version 1.

https://bundlephobia.com/package/@tiptap/core@2.0.0-beta.209

Pell is contenteditable on steroids, and it's quite good at that, which is why it's a shame it was abandoned. In my case, I required too much from an RTE and the state of the project meant I had to upgrade and have been really happy and impressed with tiptap. If your primary concern with a framework is bundle size, I question the validity of that. Except in the case of moment.js 😅

**Comment by tmikaeld** [comment]
@VaelVictus Hmm, using tiptap core with only Editor, gives a minified bundle of 239kb. A bit on the large side..

**Comment by VaelVictus** [comment]
> @VaelVictus Hmm, using tiptap 2.0.2 core with only Editor, gives a minified bundle of 239kb. A bit on the large side..

Are you saying you've tried this yourself? That would be pretty surprising. There's since been a full release of tiptap and it claims the core is now 60kb. As I recall from reading their release notes, they bundled a few things into it, so I'm not sure if that's accurate. Anyway, bundle size was already a small concern when pell was released, and now it's just a meme.

https://bundlephobia.com/package/@tiptap/core@2.0.2

**Comment by tmikaeld** [comment]
@VaelVictus Depends on where it's used though, on mobile, the 239kb is definitely painful on a slow network.

This is what I did:
```javascript
import { Editor } from '@tiptap/core';
window.Tiptap = {};
window.Tiptap.Editor = Editor;
```

Bundle with esbuild:
```
esbuild build.js --minify --bundle  --drop:console --tree-shaking=true --outfile=dist/tiptap.min.js
```

**Comment by VaelVictus** [comment]
Just doing that costs 239kb **minified?** That is surprising if it does so; Bundlephobia may not be right in this case. I'm going to be playing with tiptap v2 in about 6 weeks, so I'll definitely keep that in mind. For me, having a modern and unabandoned RTE is worth it, but the RTE is very important for what I'm doing.

**Comment by tmikaeld** [comment]
@VaelVictus Yeah, I'm as surprised as you are, it's ONLY the editor. Adding 6 extensions (including starter kit) and it's at 350kb minified, so it doesn't grow that much, but still. We're still going to use it though, there's no better headless wysiwyg out there that is this well tested.

**Comment by tmikaeld** [comment]
@VaelVictus It's not tiptap that has grown, it's [Prosemirror](https://prosemirror.net/examples/prosemirror.js) @ 254kb minified, tiptap is smaller because it uses fewer pm modules.

**Comment by gxanshu** [comment]
there site is not working as well. it returns 404
